src/PrivateRouteLogin/PrivateRouteLogin.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { Route, Redirect } from 'react-router-dom'
+
+export default function PrivateRouteLogin({component: Component, ...rest}) {
+  return (
+    <Route
+    {...rest}
+    render={(props) => {
+      if (!localStorage.getItem(process.env.REACT_APP_TOKEN_NAME)) {
+        return <Component {...props} />
+      }
+      return <Redirect to={
+        {
+          pathname: '/departments',
+          state: {
+            from: props.location
+          }
+        }
+      }
+      />
+    }}
+    />
+  )
+}
```

Comment:
no need to make login a private route. If user wants to go to login page i think we can allow him doing so

Reply:
yes, but now I don't have Logout component and if user are going to Login page, he will ask back-end for data and token - application already has token. So I've decided to make redirect to Departments page.
Maybe I'm not right, but I like this solution.

Reply:
this is fine since token gets replaced onto a new valid one. 
Consider it like sitting in different browser incognito windows simultaneously